test/recipes/30-test_evp_data/evpkdf_pbkdf2.txt

```diff
@@ -275,3 +275,39 @@ Ctrl.iter = iter:10
 Ctrl.digest = digest:sha1
 Result = KDF_CTRL_ERROR
 Reason = invalid iteration count
+
+Title = Test that a short password is blocked
+
+Availablein = fips
+FIPSversion = >=3.4.0
```

Comment:
I do not think this will go into 3.4. At least not per our stable branch policy.

Reply:
Sorry, that was a copy paste from earlier test case without thinking too much about the fact that these tests are uses for cross-version compat testing.